var/spack/repos/builtin/packages/py-protobuf/package.py

```diff
@@ -62,20 +61,18 @@ class PyProtobuf(PythonPackage):
     version("3.3.0", sha256="1cbcee2c45773f57cb6de7ee0eceb97f92b9b69c0178305509b162c0160c1f04")
     version("3.0.0", sha256="ecc40bc30f1183b418fe0ec0c90bc3b53fa1707c4205ee278c6b90479e5b6ff5")
 
-    # newer versions support python 3.11+, but we need to switch to bazel
-    depends_on("python@:3.10", type=("build", "link", "run"))
```

Comment:
@haampie which versions did you see have issues with Python 3.11? I built 4.24 and 4.23 with 3.11 without issue.